doc/README

```diff
@@ -30,21 +30,28 @@
     	% A Few Bugs in the Manual https://github.com/the3dfxdude/7kaa/issues/206 
     	% Sectioning and Text Spacing https://github.com/the3dfxdude/7kaa/issues/202
     	% Chapter Headings Misaligned https://github.com/the3dfxdude/7kaa/issues/203
+	% BUG HERE: ~ pg 11, pg 12, pg 14, pg 17, pg 18, pg 44, pg 47, pg 62, pg 64, pg 65, pg 69, pg 73, pg 93, pg 86, pg 87, pg 91 ...
```

Comment:
The purpose of BUG HERE note in the game code, is to identify something that does a thing that is not intended, even if it does work for an immediate need where fixing another bug, we can't know how to test everything, but the code has to run at least that far. So it is noted inline with the code so if someone finds a problem elsewhere, and looks around and see this, they can know that code does something actually wrong.

Using it here, I don't know what this is referring to. I only have page numbers.

Reply:
I have seen BUG HERE comments in the game code, so that is probably where I learned to use them for the manual. They are different in the manual's case tho, and if they are confusing, I can remove them.

But I will be trying to expand those page numbers with more details about the bugs as the next step to fixing them. If I remember correctly, they are just floating images mostly or collided environments. Or some issue with wrapfig and whitespace. I just want to avoid using \clearpage this time around.